capstone-report.md

```diff
@@ -20,20 +20,21 @@ name: example
 Some examples of images found in our dataset. Top row is negative (no lipohypertrophy present) and bottom row is positive (lipohypertrophy present) images where the yellow annotations indicate the exact area of the mass. 
 ```
 
-Our current dataset, as shown in Figure 2 (below), includes 218 negative images (no lipohypertrophy present) and 135 positive images (lipohypertrophy present) which is typically considered to be a very small dataset for a deep learning model. Thus, our specific data science objectives for this project include:
+Our current dataset includes 218 negative images (no lipohypertrophy present) and 135 positive images (lipohypertrophy present) as shown in {numref}`counts_df` below. Notably, this is considered to be a very small dataset for a deep learning model. 
 
-1) Use the provided dataset to develop and evaluate the efficacy of an image classification model. Some of our specific goals include:
+Our specific data science objectives for this project include:
 
-    1.1 Data Augmentations
+1. Use the provided dataset to develop and evaluate the efficacy of an image classification model. Some of our specific goals include:
+
+    1. Data Augmentation: applying random image transformations as to expland the dataset (Section 4.1)
   
-    1.2 Transfer Learning
+    2. Transfer Learning: using a model that has been pre-trained on thousands of images and using it for our dataset (Section 4.2)
   
-    1.3 Optimization of Learning
+    3. Optimization of Learning: figuring out what are the best parameters that will make the model most optimal for learning the data (Section 4.2)
   
-    1.4 Object Detection
+    4. Object Detection: training a model to detect the location of lipohypertrophy on an image (Section 4.3)
   
-2) Deploy the model for a non-technical audience.
-
+2. Deploy the model for a non-technical audience (Section 5). 
```

Comment:
I have the same comment here.. wondering if we can quick link the sections in parenthesis.